docs/ac-remote-update/how-it-works.md

```diff
@@ -0,0 +1,61 @@
+# Remote update
+
+Remote update is a capability of Agent Control that enable updating its version remotely, using Fleet Control. That includes Flux in Kubernetes. However, this feature is not available in all the supported systems yet.
+
+## Supported systems
+
+| System     | Supported |
+|------------|-----------|
+| OnHost     | ❌        |
+| Kubernetes | ✅        |
+
+### Kubernetes
+
+In Kubernetes, both the Agent Control version and Flux version can be updated. We provide a [Helm Chart to install Agent Control](https://github.com/newrelic/helm-charts/tree/master/charts/agent-control), which installs the latest version of Agent Control and Flux 2.15.0. There are some caveats.
+
+#### Caveats
+
+##### Updating Agent Control version
+
+We could receive a remote configuration with an older Agent Control version which doesn't support this feature. In that situation, the Agent Control would end up in a "non-recoverable" state. Updating it would require access to the system. Ideally, the backend should prevent that.
+
+##### Updating Flux version
+
+Flux it's a project of the [CNCF](https://www.cncf.io/) with their own roadmap. Thus, we have to be aware of breaking changes introduced in [flux2](https://github.com/fluxcd/flux2), and potential compatibility issues with the Kubernetes version that we are using. That information is usually in the changelog. For example, [v2.6.0](https://github.com/fluxcd/flux2/releases/tag/v2.6.0). **Migrations** are not automated.
+
+We could receive a remote config that installs Flux and breaks Agent Control because some breaking changed where introduced to the [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions).
+
+## How it works
+
+The high level overview of the happy path is simple:
+
+1. Fleet Control sends a new remote config with the updated version through OpAMP
+2. Agent Control receives and validates the configuration
+3. Agent Control applies the update on the system
+
+The implementation is, of course, more complex.
+
+### Kubernetes
+
+Let's start with the Agent Control update.
+
+![](./images/update-ac.png)
+
+1. **Receive Configuration**: The process begins when a new remote config is received in the `process_events` function, which then passes it to `handle_remote_config` for processing.
```

Comment:
Would it make sense to specify the remote configuration field or an example of the configuration body for AC?

Reply:
The versions are duplicated too.